test/fixtures/baseTest/go.mod

```diff
@@ -0,0 +1 @@
+module github.com/microsoft/vscode-go/gofixtures/basetest
```

Comment:
Does this assume that the user has cloned the `vscode-go` repo under `github.com/microsoft` or that doesn't matter?
Sorry, my module knowledge is not up to date :(

Reply:
Don't worry, I've struggled myself with this lately (we don't use 1.11 at work yet)

As long as they're not dependencies it doesn't matter which path they have, I believe

Reply:
Thanks @lggomez 

@jhendrixMSFT That is your understanding as well, right?

Reply:
Sorry for the delay.  This is correct, since nobody will be importing these packages the value is solely used as the identity.